ui/apps/platform/cypress/fixtures/alerts/deploymentForAlertFirstInAlerts.json

```diff
@@ -454,5 +454,15 @@
                 "gid": 4294967295
             }
         }
-    ]
+    ],
+    "serviceAccountPermissionLevel": "NONE",
+    "automountServiceAccountToken": true,
+    "hostNetwork": false,
+    "hostPid": false,
+    "hostIpc": false,
+    "runtimeClass": "",
+    "tolerations": [],
+    "ports": [],
+    "stateTimestamp": "1688993859040137",
+    "riskScore": 5.7972984
```

Comment:
These are some (but not necessarily all) required properties that have been added to the [backend's](https://github.com/stackrox/stackrox/blame/master/proto/storage/deployment.proto) `Deployment` type since these fixtures were originally created.